Assets/HoloToolkit/Speech/Scripts/TextToSpeechManager.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+using System;
+using UnityEngine;
+
+#if WINDOWS_UWP
+using Windows.Media.SpeechSynthesis;
+using Windows.Storage.Streams;
+using System.Threading.Tasks;
+#endif
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Enables text to speech using the Windows 10 <see cref="SpeechSynthesizer"/> class.
+    /// </summary>
+    /// <remarks>
+    /// <see cref="SpeechSynthesizer"/> generates speech as a <see cref="SpeechSynthesisStream"/>. 
+    /// This class converts that stream into a Unity <see cref="AudioClip"/> and plays the clip using 
+    /// the <see cref="AudioSource"/> you supply in the inspector. This allows you to position the voice 
+    /// as desired in 3D space. One recommended approach is to place the AudioSource on an empty 
+    /// GameObject that is a child of Main Camera and position it approximately 0.6 units above the 
+    /// camera. This orientation will sound similar to Cortanas speech in the OS.
+    /// </remarks>
+    public class TextToSpeechManager : Singleton<TextToSpeechManager>
```

Comment:
My only qualm with this being a Singleton is that I wouldn't be able to have multiple AudioSources playing text spatialized in different positions in my scene. I'm not sure how common this use case would actually be though, and a workaround could be just moving the AudioSource around, since I don't think having more than one playing at a time would be a common scenario. Not really an issue, just something to think about.